themes/bootstrap5/js/record.js

```diff
@@ -5,81 +5,79 @@
-    checkRequestIsValid(this, 'Hold', 'place-hold');
-  });
-  $('.checkStorageRetrievalRequest').each(function checkStorageRetrievalRequest() {
-    checkRequestIsValid(this, 'StorageRetrievalRequest', 'place-storage-retrieval');
-  });
-  $('.checkILLRequest').each(function checkILLRequest() {
-    checkRequestIsValid(this, 'ILLRequest', 'place-ill-request');
-  });
+function setUpCheckRequest(_context) {
+  let context = typeof _context === "undefined" ? document : _context;
+  context.querySelectorAll('.checkRequest').forEach(
+    (element) => checkRequestIsValid(element, 'Hold', 'place-hold')
+  );
+  context.querySelectorAll('.checkStorageRetrievalRequest').forEach(
+    (element) => checkRequestIsValid(element, 'StorageRetrievalRequest', 'place-storage-retrieval')
+  );
+  context.querySelectorAll('.checkILLRequest').forEach(
+    (element) => checkRequestIsValid(element, 'ILLRequest', 'place-ill-request')
+  );
 }
 
 function deleteRecordComment(element, recordId, recordSource, commentId) {
-  var url = VuFind.path + '/AJAX/JSON?' + $.param({ method: 'deleteRecordComment', id: commentId });
-  $.ajax({
-    dataType: 'json',
-    url: url
-  })
-    .done(function deleteCommentDone(/*response*/) {
-      $($(element).closest('.comment')[0]).remove();
-    });
+  let url = VuFind.path + '/AJAX/JSON?' + new URLSearchParams({ method: 'deleteRecordComment', id: commentId });
+  fetch(url, {
+    headers: {'Accept': 'application/json'}
+  }).then(function deleteCommentDone() {
+    let comment = element.closest('.comment');
+    comment.parentNode.removeChild(comment);
+  });
 }
 
-function refreshCommentList($target, recordId, recordSource) {
-  var url = VuFind.path + '/AJAX/JSON?' + $.param({
+function refreshCommentList(target, recordId, recordSource) {
+  let url = VuFind.path + '/AJAX/JSON?' + new URLSearchParams({
     method: 'getRecordCommentsAsHTML',
     id: recordId,
     source: recordSource
   });
-  $.ajax({
-    dataType: 'json',
-    url: url
-  })
-    .done(function refreshCommentListDone(response) {
+  fetch(url, {
+    headers: {'Accept': 'application/json'}
+  }).then(response => response.json())
+    .then(function refreshCommentListDone(response) {
       // Update HTML
-      var $commentList = $target.find('.comment-list');
-      $commentList.empty();
-      $commentList.append(VuFind.updateCspNonce(response.data.html));
-      $commentList.find('.delete').off("click").on("click", function commentRefreshDeleteClick() {
-        var commentId = $(this).attr('id').substring('recordComment'.length);
-        deleteRecordComment(this, recordId, recordSource, commentId);
-        return false;
-      });
-      $target.find('.comment-form input[type="submit"]').button('reset');
-      resetCaptcha($target);
+      let commentList = target.querySelector('.comment-list');
+      VuFind.setInnerHtml(commentList, '');
+      commentList.insertAdjacentHTML('beforeend', VuFind.updateCspNonce(response.data.html));
```

Comment:
Not sure I get this. Why do we first clear it then add adjacent html? Shouldn't a single `VuFind.setInnerHtml()`do this all?